lib/puppet/agent/locker.rb

```diff
@@ -29,11 +29,11 @@ def lock
 
   # @deprecated
   def running?
-    Puppet.deprecation_warning _(<<-ENDHEREDOC)
-Puppet::Agent::Locker.running? is deprecated as it is inherently unsafe.
-The only safe way to know if the lock is locked is to try lock and perform some
-action and then handle the LockError that may result.
-ENDHEREDOC
+    #TRANSLATORS 'Puppet::Agent::Locker.running?' is a method name and should not be translated
+    message = _('Puppet::Agent::Locker.running? is deprecated as it is inherently unsafe.')
+    message += ' ' + _('The only safe way to know if the lock is locked is to try lock and perform some'\
+                       'action and then handle the LockError that may result.')
```

Comment:
It looks like we don't want to translate `LockError` either

Reply:
fixed